src/Centreon/Infrastructure/Monitoring/ResourceRepositoryRDB.php

```diff
@@ -206,11 +206,12 @@ public function findResources(ResourceFilter $filter): array
 
         $collector = new StatementCollector();
         $request = 'SELECT SQL_CALC_FOUND_ROWS '
-            . 'resource.id, resource.type, resource.name, '
+            . 'resource.id, resource.type, resource.name, resource.action_url, resource.alias,  resource.fqdn, '
```

Comment:
```suggestion
            . 'resource.id, resource.type, resource.name, resource.alias,  resource.fqdn, '
```

Reply:
resource.action_url comes by the merge of master.

Reply:
nope, it has been removed and you wrongly merged conflicts ;) https://github.com/centreon/centreon/commit/5044c529ae05fa19fbaf168d77d3505adae0d373#diff-c2589173fdcecc9bb1272eb92e1e134059d00e73532cf9f4d7074cf0006757c7L208